main.lua

```diff
@@ -28,9 +33,11 @@ printModule = function (module, moduleApi, modTxt, identing)
     for k,mod in pairs(moduleApi.modules) do
       if (module[mod.name] == nil) then
         io.write(nextIndenting.."Module MISSING: "..modTxt.."." .. mod.name .. "\n")
+        missing = missing + 1
```

Comment:
Not correct since the function is recursive in the else case below.
The contents of the module are not checked since the module is missing. However, the counters also counts the present or missing funtions inside a module. So a missing module do not add just "1" to the missing count.